extensions-contrib/tdigestsketch/src/main/java/org/apache/druid/query/aggregation/tdigestsketch/TDigestSketchUtils.java

```diff
@@ -88,16 +86,16 @@ public static boolean matchingAggregatorFactoryExists(
   {
     // Check input for equivalence.
     final boolean inputMatches;
-    final VirtualColumn virtualInput =
-        virtualColumnRegistry.findVirtualColumns(factory.requiredFields())
+    final DruidExpression virtualInput =
```

Comment:
this pattern is done in a few places, and i think could be done more efficiently, but I avoided changing the logic at this point to reduce the number of moving parts since this PR has become a bit large